fake_fire/init.lua

```diff
@@ -326,10 +329,22 @@ minetest.register_alias("fake_fire:flint", "fake_fire:flint_and_steel")
 minetest.register_lbm({
 	name = "fake_fire:reload_particles",
 	label = "restart fire particles on reload",
-	nodenames = {"fake_fire:fancy_fire"},
+	nodenames = { "fake_fire:fancy_fire" },
 	run_at_every_load = true,
 	action = function(pos, node)
 		fire_particles_off(pos)
 		fire_particles_on(pos)
 	end
 })
+
+minetest.register_lbm({
+	name = "fake_fire:reload_particles_chimney",
+	label = "restart chimney smoke on reload",
+	nodenames = fake_fire_reload_particles_nodes,
+	run_at_every_load = true,
+	action = function(pos, node)
+		stop_smoke(pos)
+		start_fire_effects(pos, node, nil, 1)
```

Comment:
Adding a check like this would fix it:
```suggestion
		if minetest.get_meta(pos):get_int("smoky") ~= 0 then
			stop_smoke(pos)
			start_fire_effects(pos, node, nil, 1)
		end
```
For the other LBM the value to check would be `layer_1`. (if you want to change that too)

Reply:
oh, maybe this took me a while to get your point: disabled smokers will be enabled this way.
next commit incoming ...